spec/features/dashboard/show_spec.rb

```diff
@@ -3,14 +3,42 @@
 describe 'As a User' do
   describe 'When visiting the dashboard page' do
     before :each do
-      @user = User.create(name: 'Mr. Fake', email: 'fake@email.com', uid: '7', token: 'token')
+      user_data = stub_omniauth
+      user = User.find_or_create_by(uid: user_data[:uid])
```

Comment:
I believe that this'll have to be `@user` or we won't be able to reference it in the test